doc/source/overview/tutorials/alpha_s.ipynb

```diff
@@ -90,10 +90,10 @@
  ],
  "metadata": {
   "interpreter": {
-   "hash": "d848e04fb80a6ca5ca4cb83c8f9c3c87bd6b1ff7fc63b1ed0e2a9e010bdcd800"
+   "hash": "0a84ba3ac8703c04e87bc503a7d00188dfd591ad56130da93c406115a1e4a408"
   },
   "kernelspec": {
-   "display_name": "Python 3.9.7 ('dis_env')",
+   "display_name": "Python 3.10.4 ('eko-KkPVjVhh-py3.10')",
```

Comment:
We need to find a way to mute these changes (as this is a local information) - otherwise we'll get bloated ...

Reply:
That's the bad part of having notebooks inside git...

Reply:
The best you can do it's simply not to add it if you believe that you didn't apply any relevant changes. If you changed something, that add the full notebook again and that's it.

However, we can do a quick online search to see if there is any better option.